Dockerfile

```diff
@@ -1,25 +1,32 @@
 ARG DOCKER_ORG=usdotfhwastoldev
 ARG DOCKER_TAG=develop
-ARG GIT_BRANCH=develop
 FROM ${DOCKER_ORG}/carma-base:${DOCKER_TAG} as base_image
 SHELL ["/bin/bash", "-c"]
 
 ARG DEBIAN_FRONTEND="noninteractive"
+ARG VERSION
+ARG VCS_REF
+ARG BUILD_DATE
+ARG GIT_BRANCH=develop 
+# Convert ARGs to ENVs to make them available past the build stage into the container runtime.
+ENV GIT_BRANCH=$GIT_BRANCH
```

Comment:
The `GIT_BRANCH` value is not needed as an environment variable in the container runtime, so I believe we can remove this statement. Let me know if you think it would still be needed for something.